ckb-importer/src/main/java/com/hartwig/serve/ckb/json/CkbJsonDirectoryReader.java

```diff
@@ -29,24 +32,28 @@ public CkbJsonDirectoryReader(@Nullable final Integer maxFilesToRead) {
 
     @NotNull
     public List<T> read(@NotNull String dir) throws IOException {
```

Comment:
Parallelizing sped up reading from about 3 minutes to 1 minute on my laptop, which is significant when skipping hotspot resolution.